packages/chronicle/src/components/api/response-panel.module.css

```
@@ -0,0 +1,7 @@
+.panel {
+  width: 100%;
+}
+
+.panel :global([class*="code-block-module_header"]) {
```

Stylelint (17.3.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)